scripts/ci/jobs/ocp_ui_e2e_tests.py

```diff
@@ -12,7 +12,6 @@
 
 # set required test parameters
 os.environ["DEPLOY_STACKROX_VIA_OPERATOR"] = "true"
-os.environ["OPENSHIFT_CI_CLUSTER_CLAIM"] = "openshift-4"
 os.environ["ORCHESTRATOR_FLAVOR"] = "openshift"
 os.environ["ROX_POSTGRES_DATASTORE"] = "false"
```

Comment:
Maybe it's time for
```suggestion
os.environ["ROX_POSTGRES_DATASTORE"] = "true"
```
?

Also notable that the lines below claim to scale the cluster to support postgres.

Reply:
> Maybe it's time for
> 
> ?
> 
> Also notable that the lines below claim to scale the cluster to support postgres.

I'd prefer to just take care of the naming first. I need to re-examine OCP UI e2e as part of ROX-15290. 
Arguably removing that `OPENSHIFT_CI_CLUSTER_CLAIM` line is extraneous to this PR but I _know_ that is no longer applicable.